sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OpenAiController.java

```diff
@@ -24,8 +21,13 @@
 @RestController
 @SuppressWarnings("unused")
 public class OpenAiController {
-  @Autowired private OpenAiService service;
+  @Autowired private NewOpenAiService service;
 
+  /**
+   * Chat request to OpenAI
+   *
+   * @return a ResponseEntity with the response content
+   */
```

Comment:
(Question)

Was this an intentional change? If yes, please ask Charles, why he deleted the JavaDocs previously.